page_creation.rst

```diff
@@ -133,9 +133,9 @@ variable so we can render that::
         {
             $number = mt_rand(0, 100);
 
-            return $this->render('lucky/number.html.twig', array(
-                'number' => $number
-            ));
+            return $this->render('lucky/number.html.twig', [
+                'number' => $number,
+            ]);
```

Comment:
I am against making this change. Using the short array syntax in newer branches makes merging branches up a nightmare.